server/odc-service/src/main/java/com/oceanbase/odc/service/flow/ApprovalPermissionService.java

```diff
@@ -256,18 +130,27 @@ public List<UserTaskInstanceEntity> listApprovableExternalInstances() {
                 .collect(Collectors.toList());
     }
 
+
+    public Map<Long, Set<UserEntity>> getApproverByFlowInstanceIds(@NonNull Collection<Long> flowInstanceIds) {
+        return getUsersByFlowInstanceIdsAndStatus(flowInstanceIds, FlowNodeStatus.getExecutingAndFinalStatuses());
```

Comment:
prefer `return getUsersByFlowInstanceIdsAndStatus(flowInstanceIds, Collectors.sington(FlowNodeStatus.EXECUTING));`

Reply:
I need exactly 'executing' and 'final' status here for querying those approvers who are approving, **approved/rejected**, and this is where the bug is located.